src/main/java/org/fenixedu/learning/domain/executionCourse/ExecutionCourseSite.java

```diff
@@ -56,13 +58,16 @@
     public ExecutionCourseSite(ExecutionCourse executionCourse) {
         checkNotNull(executionCourse);
         setExecutionCourse(executionCourse);
-        setPublished(true);
+
         setFolder(folderForPath(PortalConfiguration.getInstance().getMenu(), "courses"));
         setSlug(on("-").join(getExecutionCourse().getSigla(), getExecutionCourse().getExternalId()));
-        setCanAdminGroup(NobodyGroup.get());
-        setCanPostGroup(NobodyGroup.get());
-        setBennu(Bennu.getInstance());
 
+        setCreationDate(new DateTime());
+        setCanAdminGroup(DynamicGroup.get("managers"));
+        setCanPostGroup(TeacherResponsibleOfExecutionCourseGroup.get(executionCourse));
+
+        setPublished(true);
+        setBennu(Bennu.getInstance());
```

Comment:
Only Line 70. Attaching the instance to Bennu should be done within the CMS. This is a bug, and should be corrected at CMS. But for now lets leave it here, because this isn't fixed yet on the other module